clients/unshaded/src/main/java/tachyon/client/FileSystemMasterClient.java

```diff
@@ -468,4 +468,29 @@ public synchronized boolean unmount(TachyonURI tachyonPath) throws IOException {
     }
     throw new IOException("Failed after " + retry + " retries.");
   }
+
+  /**
+   * Persists the given Tachyon file with the given length.
+   *
+   * @param fileId file id to persist
+   * @param length the length of the file in bytes
+   * @return true if the persist succeeded, false otherwise
+   * @throws IOException if an I/O error occurs
+   */
+  public synchronized boolean persistFile(long fileId, long length)
+      throws IOException, TachyonException {
+    int retry = 0;
+    while (!mClosed && (retry ++) <= RPC_MAX_NUM_RETRY) {
+      connect();
+      try {
+        return mClient.persistFile(fileId, length);
```

Comment:
I have a pending PR that gets rid of the persistFile RPC. We should instead use the "setState" RPC here.